napari/utils/shortcuts.py

```diff
@@ -13,20 +13,20 @@
     'napari:transpose_axes': ['Control-T'],
     'napari:toggle_grid': ['Control-G'],
     'napari:toggle_selected_visibility': ['V'],
-    'napari:activate_label_erase_mode': ['1'],
-    'napari:activate_fill_mode': ['3'],
-    'napari:activate_paint_mode': ['2'],
-    'napari:activate_label_pan_zoom_mode': ['5'],
-    'napari:activate_label_picker_mode': ['4'],
+    'napari:activate_label_erase_mode': ['1', 'E'],
+    'napari:activate_fill_mode': ['3', 'F'],
+    'napari:activate_paint_mode': ['2', 'P'],
+    'napari:activate_label_pan_zoom_mode': ['5', 'Z'],
+    'napari:activate_label_picker_mode': ['4', 'L'],
     'napari:new_label': ['M'],
     'napari:decrease_label_id': ['-'],
     'napari:increase_label_id': ['='],
     'napari:decrease_brush_size': ['['],
     'napari:increase_brush_size': [']'],
     'napari:toggle_preserve_labels': ['P'],
```

Comment:
```suggestion
    'napari:toggle_preserve_labels': ['Shift'],
```
Your P for paint has a conflict with this new one that I added  here:https://github.com/napari/napari/pull/5017
The old keybind for that was Shift, but when I registered the keybind that wasn't possible: https://github.com/napari/napari/issues/5016#issuecomment-1237416747
Now it should be possible. Maybe switch that one too?

Reply:
Otherwise, everything works great!

Reply:
Good catch, I hadn't realized there was a clash. There are way too many things in napari that start with the letter P in English...

But no, I'm not going to change it to `Shift`. That violates the ARIA Authoring Practices Guide recommendations to *not* use modifier keys as shortcuts. I imagine that's reason you were blocked from doing this before, too.

Reply:
We have a lot of these overlaps. In the labels layer alone there's "paint", "(label) picker", and "preserve labels".

I don't know what the solution is. Should "preserve labels"  be renamed to "hold labels"? Should the keybinding be `Shift` + `P` or `Alt` + `P` instead? I can't think of anything I like that seems like the obvious solution...

Reply:
I only suggested shift because that's what it was before the overhaul. I think we can set it to anything as long as users a) know it exists and b) know they can change it. 

Not at my computer: but any key on the left side of the keyboard would work for the default--right hand mouse bias. I think it's more useful as a single key binding rather than requiring a modifier.